source/_integrations/dlna_dms.markdown

```diff
@@ -34,7 +34,7 @@ The `<media_identifier>` can have one of three forms:
 - `:ObjectID`: Colon followed by a server-assigned ID for an object.
 - `?query`: Question mark followed by a query string to search for, see [DLNA ContentDirectory SearchCriteria](http://www.upnp.org/specs/av/UPnP-av-ContentDirectory-v1-Service.pdf) for the syntax. The first result found will be used.
 
-URIs generated while browsing will look like the Object ID form above. However, all three forms will work with the [media_player.play_media service](integrations/media_player/#service-media_playerplay_media).
+URIs generated while browsing will look like the Object ID form above. However, all three forms will work with the [media_player.play_media](integrations/media_player/#service-media_playerplay_media) service.
```

Comment:
What is wrong with the link? I'm not sure if I follow the fix you are trying to make?

Reply:
the word 'service' is inside the link currently, and now throws an error because of that

<img width="704" alt="Schermafbeelding 2022-02-26 om 10 45 48" src="https://user-images.githubusercontent.com/33354141/155838434-2ebedce5-74cf-4da8-9433-b82c327a480f.png">

<img width="536" alt="Schermafbeelding 2022-02-26 om 10 46 29" src="https://user-images.githubusercontent.com/33354141/155838448-9e91761e-9590-4674-8f5b-cc408c966dfa.png">

Reply:
the link
```
https://www.home-assistant.io/integrations/media_player/#service-media_playerplay_media
```
is the correct url, but in the rc version it shows like
```
https://rc.home-assistant.io/integrations/dlna_dms/integrations/media_player/#service-media_playerplay_media
```
while that isnt in the html itself. How is that done? do I set the full path instead?

Reply:
```suggestion
URIs generated while browsing will look like the Object ID form above. However, all three forms will work with the [media_player.play_media](/integrations/media_player/#service-media_playerplay_media) service.
```

Reply:
right...
thanks!